crates/ruff_linter/src/rules/ruff/snapshots/ruff_linter__rules__ruff__tests__RUF019_RUF019.py.snap

```diff
@@ -0,0 +1,42 @@
+---
+source: crates/ruff_linter/src/rules/ruff/mod.rs
+---
+RUF019.py:3:4: RUF019 [*] Unnecessary key check
+  |
+1 | d = {}
+2 | # RUF019
+3 | if "k" in d and d["k"]:
+  |    ^^^^^^^^^^^^^^^^^^^ RUF019
```

Comment:
Future improvements could be to also support other checks like:

```python
if "k" in d and d["k"] == value:
	...
```

which could be replaced with:
```python
if d.get("k") == value:
	...
```

Instead of `==`, there could be `!=`, `in`, `not in`, etc.